### PR TITLE
Make PlantUML server base address configurable

### LINKED ISSUE #15: Allow configuration of the base URL for PlantUML Server
I would like to be able to have the speed of rapid page updates without the dependency on an outside server.  It would be nice if that (like the VS Code extension), you could configure the `plantUmlServerUrl` to use a per-project configured value instead of the default plantuml.com host.

### PR DESCRIPTION
**Pull request by jikuja**
* Fixes #15

### CODE REVIEW

File: cli.collect.js [hunk 1]
```diff
@@ -259,4 +259,14 @@ module.exports = async (currentConfiguration, conf, program) => {
...
     }
+
+    if (!currentConfiguration.SERVER_BASE_ADDRESS || program.config) {
+        responses = await inquirer.prompt({
+            type: 'input',
+            name: 'serverBaseAddress',
+            message: 'Change server base address',
+            default: currentConfiguration.SERVER_BASE_ADDRESS || 'https://www.plantuml.com/plantuml/'
+        });
+        conf.set('serverBaseAddress', responses.serverBaseAddress);
```
Comment (sju66) [inline cli.collect.js]: Just change serverBaseAddress by plantUmlServerBaseAddress in order to be more understandable in the c4Builder file

File: cli.list.js [hunk 2]
```diff
@@ -32,6 +32,7 @@ Generate diagram images locally: ${currentConfiguration.GENERATE_LOCAL_IMAGES !=
 Replace diagrams with a link: ${currentConfiguration.INCLUDE_LINK_TO_DIAGRAM !== undefined ? chalk.green(currentConfiguration.INCLUDE_LINK_TO_DIAGRAM) : chalk.red('not set')}
 Place diagrams before text: ${currentConfiguration.DIAGRAMS_ON_TOP !== undefined ? chalk.green(currentConfiguration.DIAGRAMS_ON_TOP) : chalk.red('not set')}
 Charset: ${currentConfiguration.CHARSET !== undefined ? chalk.green(currentConfiguration.CHARSET) : chalk.red('not set')}
+Server base address: ${currentConfiguration.SERVER_BASE_ADDRESS !== undefined ? chalk.green(currentConfiguration.SERVER_BASE_ADDRESS) : chalk.red('not set')}
```
Comment (sju66) [inline cli.list.js]: "Server base address" => "PlantUml server base address" ?

**Review by sju66 — CHANGES_REQUESTED** [review]

**Comment by adrianvlupu** [comment]
Thanks for this. I've merged a version containing both the plantuml server url and the image format. Should have probably merged this first.

**Comment by adrianvlupu** [comment]
Fixed by https://github.com/adrianvlupu/C4-Builder/pull/29

**Pull request closed without merging**